src/pretix/static/billing/js/event.js

```diff
@@ -0,0 +1,11 @@
+document.getElementById("back-btn").addEventListener("click", function () {
+    const basePath = JSON.parse(document.getElementById('base_path').textContent);
+    const url = window.location.href
+    const organizerMatch = url.match(/organizer\/([^/]+)/);
+    const organizerSlug = organizerMatch ? organizerMatch[1] : null;
+    if(!organizerSlug) {
+        console.error('Organizer slug not found');
+        return window.location.href = `${basePath}/control/organizers/`
```

Comment:
It is weird that you put a "statement" in place of "expression". The `return` keyword, if going with something, expects an "expression". Please write a proper code.